x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml

```
@@ -118,3 +118,26 @@ query = "file where 66.0 / serial_event_id == 1"
 [[queries]]
 expected_event_ids = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 46]
 query = "process where serial_event_id + ((1 + 3) * 2 / (3 - 1)) * 2 == 54 or 70 + serial_event_id < 100"
+
+[[queries]]
+query = '''
+sequence
+  [process where serial_event_id = 1]
+  [process where serial_event_id = 2]
+'''
+expected_event_ids  = [1, 2]
+
+[[queries]]
+query = '''
+sequence
+  [process where serial_event_id=1] by unique_pid
+  [process where true] by unique_ppid'''
+expected_event_ids  = [1, 2]
+
+[[queries]]
+query = '''
+sequence
+  [process where serial_event_id<3] by unique_pid
+  [process where true] by unique_ppid
+'''
+expected_event_ids  = [1, 2, 2, 3]
```

I just started switching over to this notation for tracking results of sequences. It makes it a little easier to read each result at a time.
https://github.com/endgameinc/eql/blob/8c47601e32fac0523dd639d0b1a6d087d59814b6/eql/etc/test_queries.toml#L398-L407
```suggestion
expected_event_ids  = [1, 2,
                       2, 3]
```

---

It would be useful to have the sequences separately defined instead of one big list, for example `[[1,2][2,3]]` would clearly indicate two sequences with two items each. If the matches are sorted descending, the list can be reversed as [2,3] [1,2] a lot easier than the previous list.